Wrappers/Python/cil/optimisation/algorithms/FISTA.py

```diff
@@ -121,8 +127,20 @@ def set_up(self, initial, f, g, step_size, **kwargs):
         # set up ISTA      
         self.initial = initial
         self.x_old = initial.copy()
-        self.x = initial.copy()           
+        self.x = initial.copy()    
+        
+        if f is None:
+            f=ZeroFunction()
+            
+            if step_size is None:
+                step_size=1   
+                
+            if g is None: 
+                warnings.warn('You set both f and g to be None and thus the iterative method will not update and will remain fixed at the initial value.')
```

Comment:
do we do `warnings.warn` or `logging.warning`?

Reply:
Thanks Edo - have change this!